docs/source/acknowledge.rst

```diff
@@ -17,62 +17,49 @@ Preferred Citation
 
 Please acknowledge the use of this software within the body of your publication by copying or adapting the following:
 
-*The computational workflow in general and data management in particular for this publication was primarily supported by the signac data management framework[1].*
+*The computational workflow and data management for this publication was supported by the signac data management framework [ADRG18].*
 
-  [1] Carl S. Adorf, Paul M. Dodd, Vyas Ramasubramani, and Sharon C. Glotzer. Simple data and workflow management with the signac framework. Computational Materials Science, 146(C):220-229, 2018. `DOI:10.1016/j.commatsci.2018.01.035 <https://doi.org/10.1016/j.commatsci.2018.01.035>`__.
+.. bibliography::
+    :filter: False
 
-If you make use of signac-flow, we also strongly recommend citing the following publication:
+    signac_commat
 
-  [2] Vyas Ramasubramani, Carl S. Adorf, Paul M. Dodd, Bradley D. Dice, and Sharon C. Glotzer. signac: A Python framework for data and workflow management. Proceedings of the 17th Python in Science Conference, 152-159, 2018. `DOI:10.25080/Majora-4af1f417-016 <https://doi.org/10.25080/Majora-4af1f417-016>`__.
+A preprint of this paper is also available as `arXiv:1611.03543 <https://arxiv.org/abs/1611.03543>`__.
+
+--------
+
+If you make use of **signac-flow**, we also strongly recommend citing the following publication:
+
+.. bibliography::
+    :filter: False
+
+    signac_scipy_2018
+
+--------
+
+If your paper makes use of the **signac-flow** features for aggregation, groups, and bundling or **signac** features for synced collections, consider citing:
```

Comment:
I would remove bundling from that. This feature has been around for a while.

Reply:
Good idea.